src/e2e/java/teammates/e2e/cases/AdminHomePageE2ETest.java

```diff
@@ -26,9 +25,6 @@ public void testAll() {
         String name = "AHPUiT Instrúctör WithPlusInEmail";
         String email = "AHPUiT+++_.instr1!@gmail.tmt";
         String institute = "TEAMMATES Test Institute 1";
-        String demoCourseId = "AHPUiT____.instr1_.gma-demo";
-
-        BACKDOOR.deleteCourse(demoCourseId);
```

Comment:
While you no longer need to delete the sample course anymore, you now need to delete the account request freshly created in this test case.